runtime/service/build.gradle.kts

```diff
@@ -135,12 +136,6 @@ dependencies {
   testImplementation("software.amazon.awssdk:kms")
   testImplementation("software.amazon.awssdk:dynamodb")
 
-  runtimeOnly(project(":polaris-relational-jdbc"))
-  runtimeOnly("io.quarkus:quarkus-jdbc-postgresql") {
-    exclude(group = "org.antlr", module = "antlr4-runtime")
-    exclude(group = "org.scala-lang", module = "scala-library")
-    exclude(group = "org.scala-lang", module = "scala-reflect")
```

Comment:
Those excludes are no longer necessary?

Reply:
These were introduced to overcome the problems with `SparkIT` – but now `SparkIT` is isolated in a different module, so we can remove this.

But, this is an unrelated change, so I will create a separate PR for it:

https://github.com/apache/polaris/pull/2273